futurenhs.app/components/_pageTemplates/AdminUsersInviteTemplate/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from 'react';
+
+import { getServiceErrorDataValidationErrors } from '@services/index';
+import { getGenericFormError } from '@helpers/util/form';
+import { selectFormDefaultFields, selectFormErrors, selectFormInitialValues } from '@selectors/forms';
+import { formTypes } from '@constants/forms';
+import { FormWithErrorSummary } from '@components/FormWithErrorSummary';
+import { LayoutColumnContainer } from '@components/LayoutColumnContainer';
+import { LayoutColumn } from '@components/LayoutColumn';
+import { postSiteUserInvite } from '@services/postSiteUserInvite';
+import { FormErrors } from '@appTypes/form';
+
+import { Props } from './interfaces';
+
+/**
+ * Admin invite user template
+ */
+export const AdminUsersInviteTemplate: (props: Props) => JSX.Element = ({
+    csrfToken,
+    forms,
+    routes,
+    user,
+    contentText,
+    services = {
+        postSiteUserInvite: postSiteUserInvite
+    }
+}) => {
+
+    const [errors, setErrors] = useState(selectFormErrors(forms, formTypes.INVITE_USER));
+
+    const fields = selectFormDefaultFields(forms, formTypes.INVITE_USER);
+    const initialValues = selectFormInitialValues(forms, formTypes.INVITE_USER);
+
+    const { secondaryHeading } = contentText ?? {};
+
+    /**
+     * Client-side submission handler
+     */
+    const handleSubmit = async (formData: FormData): Promise<FormErrors> => {
+
+        try {
+
+            await services.postSiteUserInvite({ user, body: formData as any });
+
+            return Promise.resolve({});
+
+        } catch (error) {
+
+            const errors: FormErrors = getServiceErrorDataValidationErrors(error) || getGenericFormError(error);
+
+            setErrors(errors);
+
+            return Promise.resolve(errors);
+
+        }
+
+    };
+
+    /**
+     * Render
+     */
+    return (
+
+        <LayoutColumnContainer>
+            <LayoutColumn className="c-page-body">
+                <LayoutColumnContainer>
+                    <LayoutColumn tablet={8}>
+                        <FormWithErrorSummary
+                            csrfToken={csrfToken}
+                            formId={formTypes.CREATE_DISCUSSION}
+                            fields={fields}
+                            initialValues={initialValues}
+                            errors={errors}
+                            text={{
+                                errorSummary: {
+                                    body: 'There is a problem'
+                                },
+                                form: {
+                                    submitButton: 'Send invite',
+                                    cancelButton: 'Discard invite'
```

Comment:
Is the button text supposed to be capitalised? In some places we have have lowercase such as 'Save and continue' and others capitalised such as 'Upload File' and 'Create Discussion'.

Will approve but we should maybe go through them all to keep them consistent?